cas-server-support-oauth/src/main/java/org/jasig/cas/support/oauth/CentralOAuthService.java

```diff
@@ -123,14 +125,18 @@ AuthorizationCode grantAuthorizationCode(TokenType type, String clientId, String
      */
     Boolean revokeToken(Token token);
 
-    /**
-     * Revoke all Tokens associated with the specified client id, authorized by the client secret.
-     *
-     * @param clientId the client id
-     * @param clientSecret the client secret
-     * @return a Boolean status if tokens were successfully revoked.
-     */
-    Boolean revokeClientTokens(String clientId, String clientSecret);
+//    /**
+//     * Revoke all Tokens associated with the specified client id, authorized by the client secret.
+//     *
+//     * Note: This method is deprecated. Please avoid implementing it or using its implementations.
+//     *       The functionality is now performed by the `OAuth20RevokeClientTokensController`.
+//     * TODO: Remove this method and its implementations
+//     *
+//     * @param clientId the client id
+//     * @param clientSecret the client secret
+//     * @return a Boolean status if tokens were successfully revoked.
+//     */
+//    Boolean revokeClientTokens(String clientId, String clientSecret);
```

Comment:
Deprecating the `revokeClientTokens()` which is no longer used.
For future references, I decide to keep it in the code base commented out for now and for a while.